views/cypress/tests/manage-schema.spec.js

```diff
@@ -124,6 +148,10 @@
             cy.renameSelectedNode(selectors.itemForm, selectors.editItemUrl, childItemName);
         });
 
+        it('appears error on save due to notEmpty restriction', function () {
```

Comment:
Two other restriction validations are pending (formFieldOrder and languageDependant)